### PR TITLE
put data into a table

### PR DESCRIPTION
**Pull request by foodaka**
Changed the data so it is cleaner and put into a table on managebac. The command and Faria MQ mode had been updated and implemented previously.

### CODE REVIEW

File: managebac.md [hunk 1]
```diff
@@ -111,38 +111,45 @@ For example, when a new student applies to the school, they are admitted on [Ope
...
+|other_name| string| user's other name|
+|gender|string| user's gender|
+|birthday|string| user's birthday|
+|language|string| user's language|
+|password|string| account passwords
+|year_label|string| year label|
+|national_id|string| user's national id's|
+|language1 .. language4|string| user's languages|
+|nationality1 .. nationality4|string | user's nationality|
+
```
Comment (evadne) [inline managebac.md]: These lines are new, should remove

File: managebac.md [hunk 2]
```diff
@@ -111,38 +111,45 @@ For example, when a new student applies to the school, they are admitted on [Ope
...
+|nickname| string| user's nickname |
+|other_name| string| user's other name|
+|gender|string| user's gender|
+|birthday|string| user's birthday|
+|language|string| user's language|
+|password|string| account passwords
+|year_label|string| year label|
+|national_id|string| user's national id's|
+|language1 .. language4|string| user's languages|
+|nationality1 .. nationality4|string | user's nationality|
```
Comment (evadne) [inline managebac.md]: Should not skip, please list each attribute individually for clarity

File: managebac.md [hunk 3]
```diff
@@ -111,38 +111,45 @@ For example, when a new student applies to the school, they are admitted on [Ope
...
+|homeroom_advisor| | user_id or email for the advisor or admin |
+|nickname| string| user's nickname |
+|other_name| string| user's other name|
+|gender|string| user's gender|
+|birthday|string| user's birthday|
+|language|string| user's language|
+|password|string| account passwords
+|year_label|string| year label|
+|national_id|string| user's national id's|
+|language1 .. language4|string| user's languages|
```
Comment (evadne) [inline managebac.md]: Should not skip, please list each attribute individually for clarity

File: managebac.md [hunk 4]
```diff
@@ -111,38 +111,45 @@ For example, when a new student applies to the school, they are admitted on [Ope
...
-``other_name``                   - string, user's other name</br>
-``gender``                       - string, user's gender</br>
-``birthday``                     - string, user's birthday</br>
-``language``                     - string, user's language</br>
-``password``                     - string, account passwords</br>
-``year_label``                   - string, year label</br>
-``national_id``                  - string, user's national id's</br>
-``language1 .. language4``       - string, user's languages</br>
-``nationality1 .. nationality4`` - string, user's nationality</br>
+|Field |Type | |
```
Comment (evadne) [inline managebac.md]: Field / Type / Description

File: managebac.md [hunk 5]
```diff
@@ -111,38 +111,45 @@ For example, when a new student applies to the school, they are admitted on [Ope
...
+|---------------| ----------------|
+|ib_group_id |integer| ID of IB Group|
```
Comment (evadne) [inline managebac.md]: Descriptions should be written like sentences

File: managebac.md [hunk 6]
```diff
@@ -111,38 +111,45 @@ For example, when a new student applies to the school, they are admitted on [Ope
 
 ###Required Fields:
 
-``type``          - string, options: Student | Parent | Advisor</br>
-``student_id``    - integer, required only for students</br>
-``first_name``    - string, student first name</br>
-``last_name``     - string, student last name</br>
-``email``         - string, student email</br>
+| Field  | Type  | |
```
Comment (evadne) [inline managebac.md]: Field / Type / Description

File: managebac.md [hunk 7]
```diff
@@ -111,38 +111,45 @@ For example, when a new student applies to the school, they are admitted on [Ope
...
+| ------------- | ------------- |
+| type  | string| options: Student | Parent | Advisor |
```
Comment (evadne) [inline managebac.md]: Should not have the pipe character (`|`) in a table, this will mess with the layout

File: managebac.md [hunk 8]
```diff
@@ -111,38 +111,45 @@ For example, when a new student applies to the school, they are admitted on [Ope
...
+| student_id  | integer| required only for students  |
+| first_name | string| student first name |
+| last_name | string| student last name |
+| email | string| student email |
```
Comment (evadne) [inline managebac.md]: All descriptions should be sentences

File: managebac.md [hunk 9]
```diff
@@ -111,38 +111,45 @@ For example, when a new student applies to the school, they are admitted on [Ope
...
+
```
Comment (evadne) [inline managebac.md]: Remove empty line / lines

**Comment by evadne** [comment]
The commit message is not immediately clear, please re-do the message with `git commit --amend` and then force push. Write it descriptively.

http://chris.beams.io/posts/git-commit/

File: managebac.md (hunk 5 above)
Comment (foodaka) [inline managebac.md]: such as: "language is entered as a string, to notify the system of a users language" ?

Comment (evadne) [inline managebac.md]: Yes. In addition these fields usually come from some kind of Data Dictionary, so we need to investigate what values are accepted and what they mean

File: managebac.md (hunk 9 above)
Comment (foodaka) [inline managebac.md]: pushed the new changes, do i need to merge the pull request?

**Pull request closed without merging**